src/operations/hull/hullChain.js

```diff
@@ -0,0 +1,32 @@
+const flatten = require('../../utils/flatten')
+
+const union = require('../booleans/union')
+
+const hull = require('./hull')
+
+/** Create a chain of hulled geometries from the given gemetries.
+ * Essentially hull A+B, B+C, C+D, etc., then union the results.
+ * Originally "Whosa whatsis" suggested "Chain Hull"
+ * @param {...geom2} geometries - list of geometries from which to create hulls
+ * @returns {geom2} new geometry
+ *
+ * @example:
+ * let newshape = hullChain(rectangle({center: [-5,-5]}), circle({center: [0,0]}), rectangle({center: [5,5]}))
+ */
+const hullChain = (options, ...geometries) => {
+  const defaults = { closed: false }
+  let { closed } = Object.assign({}, defaults, options)
```

Comment:
I really wonder about the optional parameter
- it makes the api with an empty object at the start weird ?
- is the 'closed' aspect really needed ? (I am actually asking, not being ironic)

Reply:
Good point!

The 'closed' option just fixes the list of objects if start and end objects are different.

Question... Does the 'close' option make this function more difficutl to understand? It seems so to me.

Reply:
Hmm you could just put the same shape again at the end of the list and obtain the same result right ?
I do think it makes it harder to understand WITH the option, so yes, I think removing it is ok

Reply:
Done, removed options. I updated the test suite and the documentation as well.